runtime/centrifuge/src/weights/frame_system.rs

```diff
@@ -64,6 +64,11 @@ impl<T: frame_system::Config> frame_system::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(1))
 			.saturating_add(T::DbWeight::get().writes(2))
 	}
+
+	fn set_code() -> Weight {
+		todo!()
+	}
```

Comment:
Unfortunately, the `frame_system` bench fix is [not included](https://github.com/paritytech/cumulus/pull/2766#issuecomment-1690303367) in Polkadot v0.9.43. It is included in v1.0.0. I propose to disable benchmarks for `frame_system` in the script until we upgrade to v1.0.0. The `set_code` call can only be called via root, so I don't think it imposes risks.

WDYT @NunoAlexandre?

Reply:
Agreed 1000% 👍